### PR TITLE
Add update 42

### PR DESCRIPTION
**Pull request by stringhandler**

### CODE REVIEW

File: _updates/2020-11-25-update-42.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+class: subpage
+---
+
+After the Ridcully testnet reset, the community has been able to find and fix some critical bugs. Release v0.6.1 brings
+a number of fixes including an important fix to the process of recovering from an unsuccessful reorg. This should make the
+network more stable and prevent network segregation due to banning.
+
+## Major changes since v0.6.0
+
+- [#2431](https://github.com/tari-project/tari/pull/2431) Database refactor to remove checkpoints, fix failure on reorg and more efficient calculation of MMR roots
```
Comment (CjS77) [inline _updates/2020-11-25-update-42.md]:
```suggestion
- [#2431](https://github.com/tari-project/tari/pull/2431) Database refactor to replace MMR checkpoints with a more efficient and simpler approach; fix reorg failures
```

File: _updates/2020-11-25-update-42.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- [#2448](https://github.com/tari-project/tari/pull/2448) Add OpenSSL to Windows install, runtime
+- [#2434](https://github.com/tari-project/tari/pull/2434) Add supervisord setup notes
+- [#2439](https://github.com/tari-project/tari/pull/2439) Provide initial sync status to merge mining proxy
+- [#2377](https://github.com/tari-project/tari/pull/2377) Message malleability detect and ban
+- [#2447](https://github.com/tari-project/tari/pull/2447) Fix for xmrig powershell script
+- [#2440](https://github.com/tari-project/tari/pull/2440) Stagenet Setup Guide Corrections
+- [#2444](https://github.com/tari-project/tari/pull/2444) DHT connectivity waits for comms connectivity before starting
+- [#2427](https://github.com/tari-project/tari/pull/2427) Update merge mining runtime, README, Win install
+- [#2420](https://github.com/tari-project/tari/pull/2420) Show base node chain tip and sync status in the console wallet
+- [#2419](https://github.com/tari-project/tari/pull/2419) Add optional fee_per_gram field to console wallet Send Tx form
```
Comment (CjS77) [inline _updates/2020-11-25-update-42.md]: Does this allow variable fees in console wallet? If so, you can promote it more as a "new feature"

File: _updates/2020-11-25-update-42.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+- [#2421](https://github.com/tari-project/tari/pull/2421) Fix QR code rendering in the console wallet
+- [#2423](https://github.com/tari-project/tari/pull/2423) Plumb in the balance in the console wallet
+- [#2415](https://github.com/tari-project/tari/pull/2415) Prevent loop in peer sync by storing all peers attempted
+- [#2383](https://github.com/tari-project/tari/pull/2383) Implement daemon-mode in tari_base_node
```
Comment (CjS77) [inline _updates/2020-11-25-update-42.md]: Bump to major and add colour

**Review by CjS77 — COMMENTED** [review]
Nice. Do you mind making a few suggested edits to add some wight to the post? Otherwise looks great.

**Review by sdbondi — APPROVED** [review]
Looks good - all I can think of suggesting is touting that there are minor security improvements (specifically in #2377)

**Comment by StriderDM** [comment]
LGTM

**Review by CjS77 — APPROVED** [review]

**Pull request merged**